Open Judge System/Data/OJS.Data/Repositories/TestRunsRepository.cs

```diff
@@ -28,7 +28,7 @@ public int DeleteBySubmissionId(int submissionId)
         public override void Delete(int id)
         {
             // TODO: evaluate if this is the best solution
-            var testRun = this.Context.DbContext.ChangeTracker.Entries<TestRun>().FirstOrDefault(t => t.Property(pr => pr.Id).CurrentValue == id).Entity ?? new TestRun { Id = id };
+            var testRun = this.Context.ChangeTracker.Entries<TestRun>().FirstOrDefault(t => t.Property(pr => pr.Id).CurrentValue == id).Entity ?? new TestRun { Id = id };
```

Comment:
Shouldn't `.FirstOrDefault` and `.Entity` evaluation be on a new line? Also consider `.Entries`